src/styles/settings.scss

```diff
@@ -1,13 +1,42 @@
 $base-padding: 24px;
 
+/* COLORS NOT USED IN APP */
+
+$blue: #007bff;
+$indigo: #6610f2;
+$purple: #6f42c1;
+$pink: #e83e8c;
+$red: #dc3545;
+$orange: #fd7e14;
+$yellow: #ffc107;
+$green: #28a745;
+$teal: #20c997;
+$cyan: #17a2b8;
+$white: #fff;
+$gray: #6c757d;
+$gray-dark: #343a40;
+$primary: #d58e32;
+$secondary: #6c757d;
+$success: #28a745;
+$info: #17a2b8;
+$warning: #ffc107;
+$danger: #dc3545;
+$light: #f8f9fa;
+$dark: #343a40;
+
 $header-topbar-bg: #434343;
 $header-bg: #363636;
-
 $footer-claim-bg: #2a2a2a;
 $footer-bg: #363636;
-
 $primary: #d58e32;
-
 $text-color: #2a2a2a;
-
 $form-border-color: #292929;
+
+/* colors used in app RIGHT NOW */
+$header-topbar-bg: $header-topbar-bg;
+$header-bg: $header-bg;
```

Comment:
What is that? :O

Reply:
Good question. I don't know either but i changed it.